src/Microsoft.Deployment.DotNet.Releases/src/Microsoft.Deployment.DotNet.Releases.csproj

```diff
@@ -3,15 +3,15 @@
   <PropertyGroup>
     <MajorVersion>1</MajorVersion>
     <MinorVersion>0</MinorVersion>
-    <PreReleaseVersionLabel>preview5</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel>preview6</PreReleaseVersionLabel>
     <SuppressFinalPackageVersion>true</SuppressFinalPackageVersion>
   </PropertyGroup>
 
   <PropertyGroup>
     <IsPackable Condition="'$(TargetArchitecture)' == 'x86' OR '$(DotNetBuildFromSource)' == 'true'">true</IsPackable>
     <IsShipping>true</IsShipping>
     <PackageId>Microsoft.Deployment.DotNet.Releases</PackageId>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net7.0</TargetFrameworks>
```

Comment:
Adding net7.0 will break source-build in main/.NET 8.0 because it will add prebuilts (the 7.0 targeting packs).  This should be net8.0 or better yet it should reference the new NetCurrent Arcade property

Reply:
Also why do you think net 7.0 is needed for source-build?  Project building in source-build need to target netstandard and/or net current.

Reply:
I pushed a fix to use `$(NetCurrent)`.